pb/custom.go

```diff
@@ -0,0 +1,89 @@
+package pstore_pb
+
+import (
+	"encoding/json"
+
+	peer "github.com/libp2p/go-libp2p-peer"
+	pt "github.com/libp2p/go-libp2p-peer/test"
+	ma "github.com/multiformats/go-multiaddr"
+)
+
+type ProtoPeerID struct {
```

Comment:
We should probably comment everything here.

Reply:
Oh yeah, I missed this file.

Reply:
Will comment the structs, and add type assertions for types, so it's clear where the methods are coming from.